packages/@aws-cdk/aws-rds/lib/instance-engine.ts

```diff
@@ -0,0 +1,526 @@
+    super({
+      engineType: 'sqlserver-web',
+      parameterGroupFamilies: [
+        { engineMajorVersion: '11', parameterGroupFamily: 'sqlserver-web-11.0' },
+        { engineMajorVersion: '12', parameterGroupFamily: 'sqlserver-web-12.0' },
+        { engineMajorVersion: '13', parameterGroupFamily: 'sqlserver-web-13.0' },
+        { engineMajorVersion: '14', parameterGroupFamily: 'sqlserver-web-14.0' },
+      ],
+      engineVersion: props.version,
+    });
+  }
+}
+
+/**
+ * Properties for SQL Server Enterprise Edition instance engines.
+ * Used in {@link DatabaseInstanceEngine.sqlServerEnterpriseEdition}.
+ */
+export interface SqlServerEeInstanceEngineProps {
+  /**
+   * The exact version of the engine to use.
+   *
+   * @default - no version specified
+   */
+  readonly version?: string;
+}
+
+class SqlServerEeInstanceEngine extends SqlServerInstanceEngine {
+  constructor(props: SqlServerEeInstanceEngineProps = {}) {
+    super({
+      engineType: 'sqlserver-ee',
+      parameterGroupFamilies: [
+        { engineMajorVersion: '11', parameterGroupFamily: 'sqlserver-ee-11.0' },
+        { engineMajorVersion: '12', parameterGroupFamily: 'sqlserver-ee-12.0' },
+        { engineMajorVersion: '13', parameterGroupFamily: 'sqlserver-ee-13.0' },
+        { engineMajorVersion: '14', parameterGroupFamily: 'sqlserver-ee-14.0' },
+      ],
+      defaultInstanceType: ec2.InstanceType.of(ec2.InstanceClass.M5, ec2.InstanceSize.XLARGE),
+      engineVersion: props.version,
+    });
+  }
+}
+
+/**
+ * A database instance engine. Provides mapping to DatabaseEngine used for
+ * secret rotation.
+ */
+export class DatabaseInstanceEngine {
+  public static readonly MARIADB: IInstanceEngine = new MariaDbInstanceEngine();
+
+  public static readonly MYSQL: IInstanceEngine = new MySqlInstanceEngine();
+
+  public static readonly ORACLE_EE: IInstanceEngine = new OracleEeInstanceEngine();
+
+  public static readonly ORACLE_SE2: IInstanceEngine = new OracleSe2InstanceEngine();
+
+  public static readonly ORACLE_SE1: IInstanceEngine = new OracleSe1InstanceEngine();
+
+  public static readonly ORACLE_SE: IInstanceEngine = new OracleSeInstanceEngine();
+
+  public static readonly POSTGRES: IInstanceEngine = new PostgresInstanceEngine();
+
+  public static readonly SQL_SERVER_EE: IInstanceEngine = new SqlServerEeInstanceEngine();
+
+  public static readonly SQL_SERVER_SE: IInstanceEngine = new SqlServerSeInstanceEngine();
+
+  public static readonly SQL_SERVER_EX: IInstanceEngine = new SqlServerExInstanceEngine();
+
+  public static readonly SQL_SERVER_WEB: IInstanceEngine = new SqlServerWebInstanceEngine();
+
+  /** Creates a new MariaDB instance engine. */
+  public static mariaDb(props: MariaDbInstanceEngineProps): IInstanceEngine {
+    return new MariaDbInstanceEngine(props);
+  }
+
+  /** Creates a new MySQL instance engine. */
+  public static mySql(props: MySqlInstanceEngineProps): IInstanceEngine {
```

Comment:
Common where?

Reply:
The official client is called is called 'mysql', not 'mySql' or 'my-sql' or 'my_sql'. 
So are the official client libraries.

It would be nice to keep this consistency.

Reply:
Done.